### PR TITLE
People Update IL

### PR DESCRIPTION
**Pull request by bfossen-ce**
People Update IL

### CODE REVIEW

**Comment by jessemortenson** [comment]
I *might* be introducing a wrinkle here, but IL legislators end their terms on 1/10/23. In cases where we have redistricting/people moving offices, I have updated start/end dates to match that convention, eg:

```
roles:
- end_date: '2023-01-10'
  type: upper
  jurisdiction: ocd-jurisdiction/country:us/state:il/government
  district: '54'
- start_date: '2023-01-11'
  type: upper
  jurisdiction: ocd-jurisdiction/country:us/state:il/government
  district: '55'
```
(ex: Jason Plummer)

File: data/il/legislature/Brian-W-Stewart-bdb2b767-7c04-4a7e-941b-9734155329d6.yml [hunk 1]
```diff
@@ -11,18 +11,10 @@ roles:
   type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:il/government
   district: '89'
-- type: upper
+- end_date: '2023-01-10'
```
Comment (maxknutsen) [inline data/il/legislature/Brian-W-Stewart-bdb2b767-7c04-4a7e-941b-9734155329d6.yml:14]: @jessemortenson this person should actually be in the retired folder. They end their time in the senate on 01/10/2023 and are not serving elsewhere

File: data/il/legislature/Darren-Bailey-65768dc3-c4b1-4c8f-bb79-5da425fa59e0.yml [hunk 2]
```diff
@@ -11,7 +11,8 @@ roles:
   type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:il/government
   district: '109'
-- type: upper
+- end_date: '2023-01-10'
```
Comment (maxknutsen) [inline data/il/legislature/Darren-Bailey-65768dc3-c4b1-4c8f-bb79-5da425fa59e0.yml:14]: @jessemortenson this person should be moved to retired.

File: data/il/legislature/Avery-Bourne-2ec3bb9c-74cc-42b5-a7d6-522e73596f48.yml [hunk 3]
```diff
@@ -17,15 +17,15 @@ offices:
   voice: 217-782-8071
   fax: 217-782-1141
 - classification: district
```
Comment (maxknutsen) [inline data/il/legislature/Avery-Bourne-2ec3bb9c-74cc-42b5-a7d6-522e73596f48.yml:20]: @jessemortenson this person should be moved to retired.

File: data/il/legislature/Chris-Bos-edfe2ce9-e447-40c0-a0c4-f82300f49707.yml [hunk 4]
```diff
@@ -21,8 +21,12 @@ links:
 - url: http://ilga.gov/house/Rep.asp?GA=102&MemberID=3006
 - url: https://ilga.gov/house/Rep.asp?GA=102&MemberID=3006
   note: homepage
```
Comment (maxknutsen) [inline data/il/legislature/Chris-Bos-edfe2ce9-e447-40c0-a0c4-f82300f49707.yml:24]: @jessemortenson this person should actually be in retired.

File: data/il/legislature/David-A-Welter-05f5cd0e-23af-486c-9b7f-446db51f8ac9.yml [hunk 5]
```diff
@@ -16,14 +16,15 @@ offices:
   address: 224-N Stratton Office Building Springfield, IL 62706
   voice: 217-782-5997
 - classification: district
-  address: 1715 N. Division Street;Suite E;Morris, IL 60450
+  address: 1715 N. Division Street Suite E Morris, IL 60450
   voice: 815-416-1475
-  name: 'District Office #1'
 links:
```
Comment (maxknutsen) [inline data/il/legislature/David-A-Welter-05f5cd0e-23af-486c-9b7f-446db51f8ac9.yml:22]: @jessemortenson this person should be in retired.

File: data/il/legislature/Deanne-M-Mazzochi-1c80fb4c-e77f-4f40-ba50-9ecac95ac967.yml [hunk 6]
```diff
@@ -17,15 +17,16 @@ offices:
   voice: 217-782-1448
   fax: 217-782-1141
 - classification: district
-  address: 1 S. Cass Ave.;Westmont Centre, Suite 205;Westmont, IL 60559
+  address: 1 S. Cass Ave. Westmont Centre, Suite 205 Westmont, IL 60559
   voice: 630-852-8633
   fax: 630-852-6530
-  name: 'District Office #1'
 links:
```
Comment (maxknutsen) [inline data/il/legislature/Deanne-M-Mazzochi-1c80fb4c-e77f-4f40-ba50-9ecac95ac967.yml:24]: @jessemortenson this person should be in retired.

File: data/il/legislature/Deborah-Conroy-6e9a253e-b518-484c-be88-c4dfa7842a56.yml [hunk 7]
```diff
@@ -16,15 +16,16 @@ offices:
   address: 275-S Stratton Office Building Springfield, IL 62706
   voice: 217-782-8158
 - classification: district
-  address: 28 S. Villa Ave.;Villa Park, IL 60181
+  address: 28 S. Villa Ave. Villa Park, IL 60181
   voice: 630-415-3520
```
Comment (maxknutsen) [inline data/il/legislature/Deborah-Conroy-6e9a253e-b518-484c-be88-c4dfa7842a56.yml:21]: @jessemortenson this person should be in retired.

File: data/il/legislature/Delia-C-Ramirez-042ae70b-4d3b-47c8-a890-64abafcc104a.yml [hunk 8]
```diff
@@ -16,18 +16,21 @@ offices:
   address: 632 Capitol Building Springfield, IL 62706
   voice: 217-782-0150
 - classification: district
-  address: 3042 W. North Avenue;Chicago, IL 60647
+  address: 4053 W. Armitage Avenue Chicago, IL 60639
   voice: 773-799-8219
```
Comment (maxknutsen) [inline data/il/legislature/Delia-C-Ramirez-042ae70b-4d3b-47c8-a890-64abafcc104a.yml:21]: @jessemortenson this person should be in retired.

File: data/il/legislature/Gregory-Harris-2df3592a-e5eb-4d7b-95e8-e80aaeeedb6d.yml [hunk 9]
```diff
@@ -16,15 +16,16 @@ offices:
   voice: 217-782-3835
   fax: 217-557-6470
 - classification: district
-  address: 1967 W. Montrose;Chicago, IL 60613
+  address: 1967 W. Montrose Chicago, IL 60613
   voice: 773-348-3434
```
Comment (maxknutsen) [inline data/il/legislature/Gregory-Harris-2df3592a-e5eb-4d7b-95e8-e80aaeeedb6d.yml:21]: @jessemortenson this person should be in retired.

File: data/il/legislature/Kathleen-Willis-31c6dfe1-d3eb-4e6b-9ab1-697643fed629.yml [hunk 10]
```diff
@@ -17,15 +17,16 @@ offices:
   voice: 217-782-3374
   fax: 217-524-0448
 - classification: district
-  address: 112 N. Wolf Rd.;Northlake, IL 60164
+  address: 112 N. Wolf Rd. Northlake, IL 60164
   voice: 708-562-6970
   fax: 708-562-6974
-  name: 'District Office #1'
 links:
```
Comment (maxknutsen) [inline data/il/legislature/Kathleen-Willis-31c6dfe1-d3eb-4e6b-9ab1-697643fed629.yml:24]: @jessemortenson this person should be in retired.

File: data/il/legislature/Keith-P-Sommer-564cd0e6-1b52-42aa-bb9c-1da635cf14ba.yml [hunk 11]
```diff
@@ -17,15 +17,16 @@ offices:
   voice: 217-782-0221
   fax: 217-557-1098
 - classification: district
-  address: 121 West Jefferson;Morton, IL 61550
+  address: 121 West Jefferson Morton, IL 61550
   voice: 309-263-9242
   fax: 309-263-8187
-  name: 'District Office #1'
 links:
```
Comment (maxknutsen) [inline data/il/legislature/Keith-P-Sommer-564cd0e6-1b52-42aa-bb9c-1da635cf14ba.yml:24]: @jessemortenson move to retired.

File: data/il/legislature/Keith-R-Wheeler-2e4c0003-8421-43af-bf90-61bbcd44212b.yml [hunk 12]
```diff
@@ -16,14 +16,15 @@ offices:
   address: 200-5N Stratton Office Building Springfield, IL 62706
   voice: 217-782-1486
 - classification: district
-  address: 959 Oak Street;North Aurora, IL 60542
+  address: 959 Oak Street North Aurora, IL 60542
   voice: 630-345-3464
-  name: 'District Office #1'
 links:
```
Comment (maxknutsen) [inline data/il/legislature/Keith-R-Wheeler-2e4c0003-8421-43af-bf90-61bbcd44212b.yml:22]: @jessemortenson move to retired.

File: data/il/legislature/LaToya-Greenwood-7da55f91-6608-48ca-a06e-f5667745ac09.yml [hunk 13]
```diff
@@ -20,16 +20,13 @@ offices:
     St. Louis, IL 62201
   voice: 618-857-8124
   fax: 618-857-8116
-- classification: district
-  address: 4700 State Street;Suite 2;East St. Louis, IL 62205
-  voice: 618-646-9557
-  fax: 618-646-9558
-  name: 'District Office #1'
 links:
```
Comment (maxknutsen) [inline data/il/legislature/LaToya-Greenwood-7da55f91-6608-48ca-a06e-f5667745ac09.yml:24]: @jessemortenson move to retired.

File: data/il/legislature/Lane-Yednock-652fcf63-73c6-4983-8e61-55227ba5720b.yml [hunk 14]
```diff
@@ -16,19 +16,22 @@ offices:
   address: 240A-W Stratton Office Building Springfield, IL 62706
   voice: 217-782-0140
```
Comment (maxknutsen) [inline data/il/legislature/Lane-Yednock-652fcf63-73c6-4983-8e61-55227ba5720b.yml:17]: Not sure if this is a problem or fixable but the code at the top indicates this person is lane and it is lance.

File: data/il/legislature/Mark-Luft-147840d3-fd4d-4b21-8436-42217c540992.yml [hunk 15]
```diff
@@ -22,8 +22,12 @@ links:
 - url: http://ilga.gov/house/Rep.asp?GA=102&MemberID=3000
```
Comment (maxknutsen) [inline data/il/legislature/Mark-Luft-147840d3-fd4d-4b21-8436-42217c540992.yml:23]: @jessemortenson move to retired

File: data/il/legislature/Michael-J-Zalewski-53545928-2682-4a50-a022-68d3b335fa5f.yml [hunk 16]
```diff
@@ -16,15 +16,16 @@ offices:
   address: 229-E Stratton Office Building Springfield, IL 62706
   voice: 217-782-5280
 - classification: district
-  address: 1 Riverside Road;Suite 204 A;Riverside, IL 60546
+  address: 1 Riverside Road Suite 204 A Riverside, IL 60546
   voice: 708-442-6500
   fax: 708-442-6501
-  name: 'District Office #1'
 links:
```
Comment (maxknutsen) [inline data/il/legislature/Michael-J-Zalewski-53545928-2682-4a50-a022-68d3b335fa5f.yml:23]: @jessemortenson move to retired

File: data/il/retired/Neil-Anderson-b676c70e-0bc2-45d2-bd4c-9f3fa70bf60e.yml [hunk 17]
```diff
@@ -7,17 +7,10 @@ image: https://ilga.gov/images/members/{90CDA259-1DEA-4D18-AE97-30051E03D154}.jp
 party:
 - name: Republican
 roles:
-- type: upper
+- end_date: '2022-12-29'
+  type: upper
   jurisdiction: ocd-jurisdiction/country:us/state:il/government
```
Comment (maxknutsen) [inline data/il/retired/Neil-Anderson-b676c70e-0bc2-45d2-bd4c-9f3fa70bf60e.yml:12]: @jessemortenson move into legislature, they are not retired.

File: data/il/legislature/Sam-Yingling-ef242fba-27a0-4d87-a69e-cef12a969bb9.yml [hunk 18]
```diff
@@ -16,15 +16,16 @@ offices:
   address: 271-S Stratton Office Building Springfield, IL 62706
   voice: 217-782-7320
 - classification: district
-  address: 1919 IL Route 83;Suite 1;Round Lake Beach, IL 60073
+  address: 1919 IL Route 83 Suite 1 Round Lake Beach, IL 60073
   voice: 847-231-6262
```
Comment (maxknutsen) [inline data/il/legislature/Sam-Yingling-ef242fba-27a0-4d87-a69e-cef12a969bb9.yml:21]: @jessemortenson move to retired.

File: data/il/legislature/Sandra-Hamilton-cdd95201-aa19-4fec-ae8b-e1d8db53161d.yml [hunk 19]
```diff
@@ -0,0 +1,22 @@
+id: ocd-person/cdd95201-aa19-4fec-ae8b-e1d8db53161d
+name: Sandra Hamilton
+given_name: Sandra
+family_name: Hamilton
+email: Hamilton@ilhousegop.org
+image: https://ilga.gov/images/members/{081B04B4-59C6-4775-A840-B0DF082CB889}.jpg
+party:
+- name: Republican
```
Comment (maxknutsen) [inline data/il/legislature/Sandra-Hamilton-cdd95201-aa19-4fec-ae8b-e1d8db53161d.yml:8]: @jessemortenson move to retired

File: data/il/legislature/Tim-Butler-093bed37-c7b0-47ff-a96e-5a42859d515d.yml [hunk 20]
```diff
@@ -19,6 +19,8 @@ links:
 - url: http://ilga.gov/house/Rep.asp?GA=102&MemberID=2889
 - url: https://ilga.gov/house/Rep.asp?GA=102&MemberID=2889
   note: homepage
+- url: https://ilga.gov/house/Rep.asp?GA=103&MemberID=3095
+  note: homepage
 other_identifiers:
```
Comment (maxknutsen) [inline data/il/legislature/Tim-Butler-093bed37-c7b0-47ff-a96e-5a42859d515d.yml:25]: @jessemortenson move to retired.

File: data/il/legislature/Tom-Demmer-2873984d-5d84-4671-b896-39071e0fe851.yml [hunk 21]
```diff
@@ -16,14 +16,15 @@ offices:
   address: 314 Capitol Building Springfield, IL 62706
   voice: 217-782-0535
 - classification: district
-  address: 105 E. First Street;Suite 110;Dixon, IL 61021
+  address: 105 E. First Street Suite 110 Dixon, IL 61021
   voice: 815-561-3690
-  name: 'District Office #1'
 links:
```
Comment (maxknutsen) [inline data/il/legislature/Tom-Demmer-2873984d-5d84-4671-b896-39071e0fe851.yml:22]: @jessemortenson move to retired.

File: data/il/retired/Celina-Villanueva-013d012f-a50e-43c8-af40-f12aa852a0da.yml [hunk 22]
```diff
@@ -12,25 +12,21 @@ roles:
   jurisdiction: ocd-jurisdiction/country:us/state:il/government
   district: '21'
 - start_date: 2020-01-07
+  end_date: '2022-12-29'
   type: upper
```
Comment (maxknutsen) [inline data/il/retired/Celina-Villanueva-013d012f-a50e-43c8-af40-f12aa852a0da.yml:16]: @jessemortenson move to legislature folder.

File: data/il/retired/Jil-Tracy-be85f354-7f46-48aa-afc7-850502fda3cb.yml [hunk 23]
```diff
@@ -17,23 +17,21 @@ roles:
   type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:il/government
   district: '94'
-- type: upper
+- end_date: '2022-12-29'
+  type: upper
   jurisdiction: ocd-jurisdiction/country:us/state:il/government
```
Comment (maxknutsen) [inline data/il/retired/Jil-Tracy-be85f354-7f46-48aa-afc7-850502fda3cb.yml:22]: @jessemortenson move to legislature.

**Review by maxknutsen — COMMENTED** [review]
@NewAgeAirbender @jessemortenson @bfossen-ce IL executive and legislature done. I ignored the municipal data but can go back to it if we decide that's important. 

A lot of people either need to be moved from legislature to retired or from retired to legislature. I left comments everywhere that is the case but let me know if you have any questions.

**Review by maxknutsen — APPROVED** [review]

**Review by samanthacara — APPROVED** [review]
Very few and minor edits to be made. I did not review municipalities, I can perform a mayoral check if needed.

**Pull request merged**